docs/docs-beta/docs/dagster-plus/deployment/serverless/runtime-environment.md

```diff
@@ -5,4 +5,162 @@ sidebar_label: "Runtime environment"
 sidebar_position: 30
 ---
 
-# Serverless runtime environment
\ No newline at end of file
+With a Dagster+ Serverless deployment, you can customize the runtime environment where your code executes. You may want to:
+- [Use a different Python version](#python-version).
+- [Use a different base image](#base-image).
+- [Include data files](#data-files).
+
+Dagster uses [PEX](https://docs.pex-tool.org/) to package your code and deploy them on Docker images. You also have the option to [disable PEX-based deploys](#disable-pex) and deploy using a Docker image instead of PEX.
+
+
+## Use a different Python version \{#python-version}
+The default Python version for Dagster+ Serverless is Python 3.8. Python versions 3.9 through 3.12 are also supported. You can specify the Python version you want to use in your GitHub or GitLab workflow, or by using the `dagster-cloud` CLI.
+
+<Tabs groupId="method">
+<TabItem value="GitHub" label="GitHub">
+In your `.github/workflows/deploy.yml` file, update the `PYTHON_VERSION` environment variable with your desired Python version:
+<CodeExample filePath="dagster-plus/deployment/serverless/runtime-environment/github_python_version.yaml" language="yaml" title="Updating the Python version in deploy.yml" />
+
+</TabItem>
+<TabItem value="GitLab" label="GitLab">
+1. Open your `.gitlab-ci.yml` file. If your `.gitlab-ci.yml` looks similar to:
+    <CodeExample filePath="dagster-plus/deployment/serverless/runtime-environment/gitlab_template.yaml" language="yaml" />
+
+2. Follow the link and replace the contents of your `.gitlab-ci.yml` with the YAML document at the link address.
```

Comment:
I'm not sure I follow these two steps? If your the gitlab-ci file looks like what you're linking too... why would would replace it? (Also you link to the dbt CI yaml?)

Are you trying to say is if their CI looks like the [serverless legacy one](https://github.com/dagster-io/dagster-cloud-action/blob/main/gitlab/serverless-legacy-ci.yml), they should update it to the [new one](https://github.com/dagster-io/dagster-cloud-action/blob/main/gitlab/serverless-ci.yml)?

Reply:
Like the file might literally say 
```
include:
  remote: https://raw.githubusercontent.com/dagster-io/dagster-cloud-action/v0.1.29/gitlab/dbt/serverless-ci-dbt.yml 
```

https://dagsterlabs.slack.com/archives/C03A0D72A6T/p1725042971176309?thread_ts=1725036052.384899&cid=C03A0D72A6T

Reply:
Oh I see... never mind then

Reply:
@maximearmstrong is going to fix this, so that the full file gets copied, rather than writing a reference to the template. So this part of the doc can be updated when that is merged

Reply:
I've opened a draft PR [here](https://github.com/dagster-io/internal/pull/11265) - still need to test everything though